test/binCases/stats/single-config/test.js

```diff
@@ -0,0 +1,30 @@
+/*globals it*/
+module.exports = function validateBinExec(cmd, args, opts) {
```

Comment:
`test/BinTestCases.test.js` should take care of execution.

This file should be passed exitcode and stdout + stderr. It should throw an expection when it's incorrect.

Example for a test case:

#### test.opts

``` text
--help
```

#### test.js

``` js
module.exports = (code, stdout, stderr) => {
  code.should.be.eql(0);
  stdout.should.startWith("webpack");
  stderr.should.be.eql("");
  stdout.should.contain("--output-path");
  // ...
};
```